### PR TITLE
fixup release process

### PR DESCRIPTION
**Pull request by dragonsinth**
I noticed a couple things while going through the release process for v1.8.7.

### CODE REVIEW

File: releasing/README.md [hunk 1]
```diff
@@ -53,22 +53,7 @@ This will create the release in GitHub with provisional release notes that just
...
-docker build -t fullstorydev/grpcurl:v2.3.4-alpine --target alpine .
-# now that we have it built, push to Docker Hub
-docker push fullstorydev/grpcurl:v2.3.4
-docker push fullstorydev/grpcurl:v2.3.4-alpine
-# push "latest" tag, too
-docker tag fullstorydev/grpcurl:v2.3.4 fullstorydev/grpcurl:latest
-docker tag fullstorydev/grpcurl:v2.3.4-alpine fullstorydev/grpcurl:latest-alpine
-docker push fullstorydev/grpcurl:latest
-docker push fullstorydev/grpcurl:latest-alpine
-```
```
Comment (dragonsinth) [inline releasing/README.md:71]: this is likely to be ever-stale; better to just refer to the actual script

File: releasing/do-release.sh [hunk 2]
```diff
@@ -50,8 +50,7 @@ export DOCKER_CLI_EXPERIMENTAL=enabled
 $PREFIX docker buildx create --use --name multiarch-builder --node multiarch-builder0
 # push to docker hub, both the given version as a tag and for "latest" tag
 $PREFIX docker buildx build --platform linux/amd64,linux/s390x,linux/arm64,linux/ppc64le --tag fullstorydev/grpcurl:${VERSION} --tag fullstorydev/grpcurl:latest --push --progress plain --no-cache .
-rm VERSION
```
Comment (dragonsinth) [inline releasing/do-release.sh:53]: copy-pasta

**Review by gpassini — APPROVED** [review]
LGTM!

**Comment by gpassini** [comment]
The [build is green](https://app.circleci.com/pipelines/github/fullstorydev/grpcurl/79/workflows/e7923a47-d2f7-4d53-b95e-9fbb3ea7b910). We need to figure out why GitHub is not reporting the correct status, but in the meanwhile I think we can already merge this PR.

**Pull request merged**